bin/start-chronos.bash

```diff
@@ -1,27 +1,39 @@
 #!/bin/bash
-set -o errexit -o nounset -o pipefail
+set -o errexit -o pipefail
 
-chronos_home="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd -P )"
-echo "Chronos home set to $chronos_home"
-export JAVA_LIBRARY_PATH="/usr/local/lib:/lib:/usr/lib"
-export LD_LIBRARY_PATH="${LD_LIBRARY_PATH:-/lib}"
-export LD_LIBRARY_PATH="$JAVA_LIBRARY_PATH:$LD_LIBRARY_PATH"
+# allow overriding ENTRYPOINT
+# see https://github.com/docker-library/official-images/issues/692
+if [ -z "$1" ] || [ "${1:0:1}" == '-' ]; then
+  set -o errexit -o nounset -o pipefail
+  flags=( "$@" )
 
-flags=( "$@" )
```

Comment:
Why are we setting the heap here? If so, it needs to be configurable with a sensible default.